ark/segmentation/marker_quantification.py

```diff
@@ -157,6 +184,10 @@ def create_marker_count_matrices(segmentation_labels, image_data, nuclear_counts
         split_large_nuclei (bool):
             boolean flag to determine whether nuclei which are larger than their assigned cell
             will get split into two different nuclear objects
+        extraction (str):
+            extraction function used to compute marker counts. default is 'total_intensity'
```

Comment:
If default is defined in function def, don't need to be put default in docstring as well